File: android_project/app/src/main/java/com/example/projetws/beans/Etudiant.java
package com.example.projetws.beans;

public class Etudiant {
    private int id;
    private String nom;
    private String prenom;
    private String ville;
    private String sexe;

    public Etudiant() {}

    public Etudiant(int id, String nom, String prenom, String ville, String sexe) {
        this.id     = id;
        this.nom    = nom;
        this.prenom = prenom;
        this.ville  = ville;
        this.sexe   = sexe;
    }

    public int getId()        { return id; }
    public String getNom()    { return nom; }
    public String getPrenom() { return prenom; }
    public String getVille()  { return ville; }
    public String getSexe()   { return sexe; }

    public void setId(int id)           { this.id = id; }
    public void setNom(String nom)      { this.nom = nom; }
    public void setPrenom(String p)     { this.prenom = p; }
    public void setVille(String v)      { this.ville = v; }
    public void setSexe(String s)       { this.sexe = s; }

    @Override
    public String toString() {
        return "Etudiant{id=" + id + ", nom='" + nom + "', prenom='" + prenom +
               "', ville='" + ville + "', sexe='" + sexe + "'}";
    }
}

File: php_project/classes/Etudiant.php
<?php
class Etudiant {
    private $id;
    private $nom;
    private $prenom;
    private $ville;
    private $sexe;

    public function __construct($id, $nom, $prenom, $ville, $sexe) {
        $this->id     = $id;
        $this->nom    = $nom;
        $this->prenom = $prenom;
        $this->ville  = $ville;
        $this->sexe   = $sexe;
    }

    public function getId()     { return $this->id; }
    public function getNom()    { return $this->nom; }
    public function getPrenom() { return $this->prenom; }
    public function getVille()  { return $this->ville; }
    public function getSexe()   { return $this->sexe; }

    public function setId($id)         { $this->id = $id; }
    public function setNom($nom)       { $this->nom = $nom; }
    public function setPrenom($prenom) { $this->prenom = $prenom; }
    public function setVille($ville)   { $this->ville = $ville; }
    public function setSexe($sexe)     { $this->sexe = $sexe; }
}
?>
